test/sharness/t0090-get.sh

```diff
@@ -54,7 +54,7 @@ test_expect_success "ipfs get -a output looks good" '
 test_expect_success "ipfs get -a archive output is valid" '
   tar -xf "$HASH".tar &&
   test_cmp $HASH data &&
-  rm $HASH.tar &&
+  rm "$HASH".tar &&
```

Comment:
quote variables everywhere, that's probably what's going on. you have a ton more vars unquoted.